integration-tests/constants/dynamoDBItemDetails.ts

```diff
@@ -18,7 +20,7 @@ export const dynamoDBItemDataPathAndPIITypes: DynamoDBItem = {
       },
       {
         id: 6202354485660,
-        value: 'txma-team2-ticf-analyst-dev@test.gov.uk'
+        value: recipientEmail
```

Comment:
Why not just take this from the constant you already have, why do you need another variable `recipientEmail` set to the same value?

Reply:
Good spot. Done.